features/default/configuration.feature

```diff
@@ -24,4 +24,32 @@ Feature: Configuration overrides
     * the trace payload field "resourceSpans.0.resource" string attribute "service.name" equals "com.bugsnag.ServiceNameScenario"
     * the trace payload field "resourceSpans.0.resource" string attribute "service.version" equals "10.0"
     * the trace payload field "resourceSpans.0.resource" string attribute "telemetry.sdk.name" equals "bugsnag.performance.cocoa"
-    * the trace payload field "resourceSpans.0.resource" string attribute "telemetry.sdk.version" matches the regex "[0-9]\.[0-9]\.[0-9]"
\ No newline at end of file
+    * the trace payload field "resourceSpans.0.resource" string attribute "telemetry.sdk.version" matches the regex "[0-9]\.[0-9]\.[0-9]"
+
+  Scenario: Fixed sampling probability
+    Given I set the sampling probability for the next traces to "0"
```

Comment:
Perhaps a scenario confirming that no spans are sent the other way around too?

Reply:
We already have that in a separate file: https://github.com/bugsnag/bugsnag-cocoa-performance/blob/94c6790816b62458db15e6489ffc889b1082a9d1/features/default/initial-p-value.feature#L3

Reply:
That's just if the dynamic rate is 0 or 1, not if the new fixed configuration is 0. I think it would help just to ensure the rate is applied from the start.